### DISCUSSION TITLE
Release 4.0?

### DISCUSSION THREAD

**Discussion by jonkoops** [auth0/jwt-decode#217] [General]
I personally cannot think of anything that still needs to be done for the 4.0 release. Should we go ahead and publish a stable version @frederikprijck?

**Comment by frederikprijck** [comment]
I have internally kicked of the process for this. Our security team will do a review, once they give a green light we are good to go.

**Reply by jonkoops** [reply]
Great! Looking forward to the new release!

**Comment by frederikprijck** [comment]
The release is out! https://github.com/auth0/jwt-decode/discussions/233

**Reply by jonkoops** [reply]
Awesome, thanks for the collaboration @frederikprijck!

**Comment by jonkoops** [comment]
@frederikprijck Looks like the release notes are missing some information on how to migrate from the default to the named exports. You might want to add some instructions like:

### Migration 🚚

The `jwtDecode` function is now no longer the default export, and is instead provided as a named export. Make sure to update your code in places where you are importing this function:

```diff
-import jwtDecodefrom "jwt-decode";
+import { jwtDecode } from "jwt-decode";
```

**Reply by frederikprijck** [reply]
Thanks, addressed it https://github.com/auth0/jwt-decode/pull/234/files
